src/lib/evil/evil_time.c

```diff
@@ -602,3 +603,29 @@ strptime(const char *buf, const char *fmt, struct tm *tm)
 
    return (char *)bp;
 }
+
+
+#ifdef _MSC_VER
+
+EAPI int 
+gettimeofday(struct timeval * tp, struct timezone * tzp)
+   {
+      // This number is the number of 100 nanosecond intervals since 00:00:00
+      // 01/01/1601 (UTC) (Windows way) until 00:00:00 01/01/1970 (POSIX way)
+      static const uint64_t EPOCH = ((uint64_t) 116444736000000000ULL);
+
+      SYSTEMTIME system_time;
+      FILETIME file_time;
+      uint64_t time;
+
+      GetSystemTime(&system_time);
+      SystemTimeToFileTime( &system_time, &file_time );
+      time =  ((uint64_t)file_time.dwLowDateTime )      ;
+      time += ((uint64_t)file_time.dwHighDateTime) << 32;
+
+      tp->tv_sec  = (long) ((time - EPOCH) / 10000000L);
+      tp->tv_usec = (long) (system_time.wMilliseconds * 1000);
+      return 0;
+   }
```

Comment:
```suggestion
{
   // This number is the number of 100 nanosecond intervals since 00:00:00
   // 01/01/1601 (UTC) (Windows way) until 00:00:00 01/01/1970 (POSIX way)
   static const uint64_t EPOCH = ((uint64_t) 116444736000000000ULL);

   SYSTEMTIME system_time;
   FILETIME file_time;
   uint64_t time;

   GetSystemTime(&system_time);
   SystemTimeToFileTime( &system_time, &file_time );
   time =  ((uint64_t)file_time.dwLowDateTime )      ;
   time += ((uint64_t)file_time.dwHighDateTime) << 32;

   tp->tv_sec  = (long) ((time - EPOCH) / 10000000L);
   tp->tv_usec = (long) (system_time.wMilliseconds * 1000);
   return 0;
}
```